requirements/requirements.in

```diff
@@ -1 +1,2 @@
 Flask>=2.0.2
+m2r2
```

Comment:
You should specify a (minimal) version for this requirement and update requirements.txt file (using pip-compile) to track higher-orders dependencies.